website/src/pages/index.js

```diff
@@ -16,14 +16,13 @@
 
 const React = require('react')
 
-const HomeSplash = require(`${process.cwd()}` + `/core/HomeSplash`)
+// const HomeSplash = require(`${process.cwd()}` + `/core/HomeSplash`)
 
-const Container = require('../../../../react-bootstrap/Container')
-const Button = require('../../../../react-bootstrap/Button')
-const Row = require('../../../../react-bootstrap/Row')
-const Col = require('../../../../react-bootstrap/Col')
-const Image = require('../../../../react-bootstrap/Image')
-const translate = require('../../server/translate').translate
+const Container = require('react-bootstrap/Container')
+const Button = require('react-bootstrap/Button')
+const Row = require('react-bootstrap/Row')
+const Col = require('react-bootstrap/Col')
+const Image = require('react-bootstrap/Image')
```

Comment:
you should rather use ES6 imports/exports

or you'd have to use

const Container = require('react-bootstrap/Container').default